File: GoblinVisual/GoblinVisual/GoblinVisual/Requetes/OuvrirCompte.cs
/** @author Max
*/

using Requete


class OuvrirCompte : Requete {

	
	public override bool doitTraiter(Receptionniste r) {
		return true;
	}
	
	public override bool doitTraiter(Coffrier c) {
		return true;
	}

	public override bool doitTraiter(Tamponneur t) {
		return true;
	}
	
	public override bool doitTraiter(Chef c) {
		return true;
	}
}

File: Java/Requetes/OuvrirCompte.java
/** @author Max
*/



class OuvrirCompte extends Requete {

	@Override
	public bool doitTraiter(Receptionniste r) {
		return true;
	}

	@Override
	public bool doitTraiter(Coffrier c) {
		return true;
	}

	@Override
	public bool doitTraiter(Tamponeur t) {
		return true;
	}

	@Override
	public bool doitTraiter(Chef c) {
		return true;
	}
}
